universum/modules/output/output.py

```diff
@@ -72,3 +61,11 @@ def log_stderr(self, line):
 
     def log_shell_output(self, line):
         self.driver.log_shell_output(line)
+
+
+class HasOutput(Module):
+    out_factory: Dependency[Output] = Dependency(Output)
```

Comment:
Do you have to add annotation here?

Reply:
not really, I just add them everywhere - better safe than sorry
I can remove this one, if you want

Reply:
I prefer to make sure that the inferred type is correct during development, but to remove repetition after that (DRY).